dbt/include/snowflake/macros/adapters.sql

```diff
@@ -73,7 +73,7 @@
   {% for _ in range(0, max_iter) %}
 
       {%- set paginated_sql -%}
-         show terse objects in {{ schema_relation.database }}.{{ schema_relation.schema }} limit {{ max_results_per_iter }} from '{{ watermark.table_name }}'
```

Comment:
At the risk of future community contributors asking why we don't just have the terse keyword in here, should we leave a comment or do we just handle that in triage?

Reply:
I think handle it in triage. If they try to use it, the integration tests will fail because the cache will think there's no dynamic tables.